gxa/src/test/java/uk/ac/ebi/atlas/bioentity/GenePageControllerWIT.java

```diff
@@ -45,14 +43,4 @@ public void geneNotFound() throws Exception {
                 .andExpect(view().name("error-page")); // Can be a view name
     }
 
-    @Test
-    public void geneIdFieldIsPresent() throws Exception {
-        String geneId = solrUtils.fetchRandomExpressionAtlasGene();
-
-        this.mockMvc.perform(get("/genes/"+geneId))
-                .andExpect(status().is(200))
-                .andExpect(view().name("search-results"))
-                .andExpect(model().attributeExists("gene_id"));
-    }
-
```

Comment:
You can bring the test back but replacing `gene_id` with `identifier`. The purpose of the test is to ensure that the model contains the necessary attributes that the JSP needs to render the info card.